corehq/apps/enterprise/enterprise.py

```diff
@@ -430,47 +429,18 @@ def total_for_domain(self, domain_obj):
 
         return query.count()
 
-    def create_count_subquery(self, **kwargs):
-        return Coalesce(
-            Subquery(
-                SMS.objects.filter(
-                    domain=OuterRef('domain'),
-                    date__gte=self.datespan.startdate,
-                    date__lt=self.datespan.enddate_adjusted,
-                    **kwargs
-                )
-                .values('domain')
-                .annotate(count=Count('pk'))
-                .values('count')
-            ),
-            0
-        )
-
     @property
     def headers(self):
         headers = [_('Project Space Name'), _('# Sent'), _('# Received'), _('# Errors')]
 
         return headers
 
     def rows_for_domain(self, domain_obj):
-        sent_subquery = self.create_count_subquery(direction=OUTGOING, processed=True)
-        received_subquery = self.create_count_subquery(direction=INCOMING, processed=True)
-        error_subquery = self.create_count_subquery(error=True)
+        results = SMS.objects.filter(domain=domain_obj.name) \
+            .values('direction', 'error').annotate(direction_count=Count('pk'))
 
-        query = (
-            SMS.objects
-            .filter(
-                domain=domain_obj.name,
-            )
-            .values('domain').distinct()
-            .annotate(sent_count=sent_subquery)
-            .annotate(received_count=received_subquery)
-            .annotate(error_count=error_subquery)
-            .values_list('domain', 'sent_count', 'received_count', 'error_count')
-        )
+        num_sent = sum([result['direction_count'] for result in results if result['direction'] == OUTGOING])
```

Comment:
Should we only include sms that is succesfully sent and received? If we don't charge them, then we should not include errored sms in `num_sent` and `num_received`

Reply:
`total_for_domain` should also align with the number in report, when people sum up num_sent for all domains

Reply:
Addressed in 1adcaf384f41033b97d3377f9185ae91c9a4562a